schemas/rum/_action-child-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "_action-child-schema.json",
+  "title": "ActionChildProperties",
+  "type": "object",
+  "description": "Schema of all properties of events that can have parent actions",
+  "properties": {
+    "action": {
+      "type": "object",
+      "description": "Action properties",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "description": "UUID of the action",
+          "readOnly": true,
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
+            },
+            {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
+              }
+            }
+          ]
+        }
+      },
+      "readOnly": true
+    }
```

Comment:
I agree with @maxep - the `oneOf` used here will be laborious to generate code in strongly typed languages. In Swift, we'd have to extend our generator and represent this with `enum`, sort of:
```swift
let id: SingleOrMultipleUUIDs

enum SingleOrMultipleUUIDs {
   case single(UUID)
   case multiple([UUID])
}
```
This notion is not something we currently support as we're not using general-purpose 3rd party generator (instead, we had to made our own). 

Is there a possibility to represent `action.id` in another way, possibly using existing schema constructs (e.g. only array, where single element would mean a single `id`)? Otherwise, we can do the work on our side, but as it requires additional time, it's IMO viable to consider alternative (if there is any and it can be correct).

Reply:
We have to keep the previous typing for retrocompatibility, but in SDKs it is fine to always send an array.

Reply:
Oh, I see 👌. So single `id` is coming from previous version of the schema and array is a new requirement 😥. Seems indeed that we don't have an easy option for that // cc @maxep